config/locales/en.yml

```diff
@@ -1,4 +1,13 @@
 en:
+  books:
+    index:
+      add_new_book_link: "Add a new book"
+      header: "Library Catalog"
+
+  labels:
+    title: "Title:"
+    author: "Author:"
```

Comment:
I think you should use just one or the other of these ones: https://github.com/margaretoti/library-network/pull/1/files#diff-596374a12ef429714137455430e6b4c4R5